tests/php/Controllers/Root/RootController/RootControllerTopActionTest.php

```diff
@@ -443,4 +443,21 @@ public function testTopActionIsTabbarStatusEqualHomeWhenIsDevParam()
             'error:アサイン情報としてtabbarStatusはhomeではない'
         );
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)